docs/requirements.rst

```diff
@@ -72,3 +74,19 @@ Additional Suggested Software
 
 - You may want to use autossh, available in various package management
   systems
+- If you are using systemd, sshuttle can notify it when the connection to
+  the remote end is established and the firewall rules are installed
+
+.. code-block:: ini
+   :emphasize-lines: 6
```

Comment:
Pushed a new commit explaining that the the process start-up type for sshuttle service unit should be set to notify for this feature to work. Thanks for reviewing @brianmay. Let me know if you have any other observations. Thanks!